operations/image_management/Configure_IMS_to_validate_rpms.md

```diff
@@ -88,16 +211,85 @@ Configuring the Image Management Service (IMS) to validate the GPG signatures of
     Dockerfile  entrypoint.sh  my-signing-key.asc
     ```
 
+1. (`ncn-mw#`) (CSM 1.5.0 and later) Verify that the following file is in the temporary directory.
```

Comment:
Since this is going into the CSM 1.6 docs, isn't the "CSM 1.5.0 and later" part unnecessary?